scripts/fornax2.sh

```diff
@@ -0,0 +1,88 @@
+#! /bin/bash
+
+set -x
```

Comment:
seems like this script has lots of common commands with fornax1.sh. please group these common commands into a function to prevent such repetition.